sechub-scan/src/main/java/com/mercedesbenz/sechub/domain/scan/log/ProjectScanLog.java

```diff
@@ -58,6 +58,8 @@
     public static final String STATUS_FAILED = "FAILED";
     public static final String STATUS_ABANDONED = "ABANDONDED";
 
+    public static final String QUERY_DELETE_LOGS_OLDER_THAN = "DELETE FROM ProjectScanLog l WHERE l." + PROPERTY_STARTED + " <:cleanTimeStamp";;
```

Comment:
```suggestion
    public static final String QUERY_DELETE_LOGS_OLDER_THAN = "DELETE FROM ProjectScanLog l WHERE l." + PROPERTY_STARTED + " < :cleanTimeStamp";
```

Furthermore, the difference between the letter `l` and `1` is difficult to identify. To go around this issue, I would prefer `log`. It would then be `ProjectScanLog log WHERE log`…